app/uk/gov/hmrc/rasapi/repository/RasChunksRepository.scala

```diff
@@ -33,17 +33,19 @@ class RasChunksRepository @Inject()(
                                    ) extends ReactiveRepository[Chunks, BSONObjectID](
     collectionName = "resultsFiles.chunks",
     mongo = mongoComponent.mongoConnector.db,
-    domainFormat = Chunks.format){
+    domainFormat = Chunks.format) {
+
+  val log: Logger = Logger(getClass)
```

Comment:
is this different to `extends Logging`?

Reply:
just realised you did this because of the Logger in ReactiveRepository, disregard this comment